### DISCUSSION TITLE
Gridsquare question

### DISCUSSION THREAD

**Discussion by KS5Y** [wavelog/wavelog#1179] [General]
Hi Folks,
Quick question, I'm having trouble finding a way to populate grid squares in QSOs imported via ADIF, is there a way to do that? 
For example, last contest I've got +300 qsos but my logging software is not adding the gridsquare, once I uploaded the adif file to wavelog I could add it manually but when the number of QSOs is  big it doesn't make sense.

Thank you

**Comment by phl0** [comment]
There is an "update from callbook" function which would populate grids. But this may not be the grids the op operated from. Especially for POTA activators or SAT rovers for example. Where would you get the grid from if it is not in the log?

**Comment by HB9HIL** [comment]
If you want to use the "Update from Callbook" Feature which @phl0 mentioned above:

1. Make sure you have a callbook configured. If you especially want the gridsquares you should configure `qrz` in your config.php as callbook. Be aware that you must have a qrz.com **XML Subscription** to get the gridsquares. 
2. Go to Logbook > Advanced. Set filters, results amount and station locations to your needs and hit search. Then you can simply mark all at once in the upper left corner of the table. 
3. Click the "Actions" button and you should see the "Update from Callbook" at the left.

**Comment by KS5Y** [comment]
Awesome! thanks for the details, I was able to pull the info from qrz.com.
73
